.github/scripts/userdata.sh

```diff
@@ -154,7 +154,7 @@ then
 sed -i "s/^echo \"cluster.name.*/echo \"cluster.name \: odfe-$ODFE_VER-$1-noauth\" \>\> \/etc\/elasticsearch\/elasticsearch.yml/g" $REPO_ROOT/userdata_$1.sh
 sed -i "/echo \"network.host/a echo \"opendistro_security.disabled: true\" \>\> \/etc\/elasticsearch\/elasticsearch.yml" $REPO_ROOT/userdata_$1.sh
 cat <<- EOF >> userdata_$1.sh
-sudo rm -rf /usr/share/kibana/plugins/opendistro_security
+sudo rm -rf /usr/share/kibana/plugins/opendistroSecurityKibana
 sudo sed -i /^opendistro_security/d /etc/kibana/kibana.yml
```

Comment:
This has to be changed to sed -i /^opendistroSecurityKibana/d /etc/kibana/kibana.yml , as opendistro_security will no longer exist in the file.

Reply:
The settings have not changed. Just the plugin name